pallets/handles/README.md

```diff
@@ -7,25 +7,56 @@ Creates human-readable, homoglyph-attack resistant handles for MSAs.
 Provides MSAs with an optional, but unique handle.
 
 A handle consists of:
-- **Base Handle:** The user's chosen handle. It is *not* guaranteed to be unique without the suffix. It is linked to a normalized version for Handle to MSA Id resolution. See [UTF-8 Support](#utf-8-support) and [Homoglyph Attack Resistence](#homoglyph-attack-resistence) below.
+- **Base Handle:** The user's chosen handle. It is *not* guaranteed to be unique without the suffix. It is linked to a normalized version for Handle to MSA Id resolution. See [Normalization Details](#normalization-details) below.
 - **Suffix:** A suffix is a unique numeric value appended to a user's base handle to make it unique.
```

Comment:
"The suffix is a numeric value appended to the user's base handle to ensure the display handle (base handle + suffix) is unique."

Reply:
```suggestion
- **Suffix:** The suffix is a numeric value appended to the user's base handle to ensure the display handle (base handle + suffix) is unique.
```